CHANGELOG.md

```diff
@@ -5,40 +5,66 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.0.0] (unreleased)
+## LeanCode.ContractsGenerator
 
-### Added
+### [2.0.0]
 
-- `LeanCode.Contracts` package with basic contracts definition,
-- `Binary` type,
-- `Operation` type,
-- Basic docs,
-- Add `dotnet contracts-generate` global tool,
+#### Added
+
+- Support for `Binary` type,
+- Support for `Operation` type,
+- Support for `struct` members,
+- Support for attributes on Enum members (`EnumValue`),
+- `dotnet contracts-generate` global tool,
 - Allow running the generator in check-only mode,
 - _Invalid type_ analyzer,
-- Enum members (`EnumValue`) have attributes,
-- Support for `struct` members.
+- Basic docs.
 
-### Breaking changes
+#### Breaking changes
 
 - The generator depends on `LeanCode.Contracts` instead of `LeanCode.CQRS` package,
 - `Date`, `Time` and `DateTime` known-types are gone,
 - The tool targets both .NET 6 and .NET 7 now.
```

Comment:
Is this really a braking change?

Reply:
Unfortunately - yes. The behavior changed slightly. :(

Reply:
Doesn't that mean that contracts v1.1.1, which also got this change, should actually be v2.0.0? :sweat_smile:

Reply:
This only affects generator, not contracts. Plus. Contracts 1.1 were introduced with v2 of the generator (so they lag slightly).